external/dear_imgui/CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/backends)
 # Create static library
 add_library(imgui STATIC ${IMGUI_SOURCES})
 
-if (APPLE OR LINUX OR UNIX)
-    target_include_directories(imgui PRIVATE ${SDL2_INCLUDE_DIRS})
+if (APPLE OR LINUX)
```

Comment:
Same here `if (APPLE OR LINUX OR UNIX)` to work for some reason.

Reply:
Fixed.
I also wanted to ask what version of clang-format checks formatting?
`sudo apt-get install -y clang-format`

Because I have tried on different versions of clang-format and it gives different results compare with your check.

Reply:
Specifically the problem with
`AlignTrailingComments: true`
Because its syntax is already deprecated since clang-format 16.

Reply:
Yeah, lets remove that from the clang-format file